auraed/src/logging/stream_logger.rs

```diff
@@ -41,6 +41,7 @@ pub struct StreamLogger {
 }
 
 impl StreamLogger {
+    #[allow(dead_code)]
```

Comment:
can we "allow(unused)" instead? unused suggests "i'm going to use this later" .. dead code suggests "this is no longer needed but i'm scared to delete it".